monitoring/tools/s2_gap_filler.py

```diff
@@ -165,12 +163,18 @@ def publish_message(files: list, queue_name: str, slack_url: str = None) -> str:
 def cli(
     idx: int,
     max_workers: int = 2,
-    limit: int = None,
     sync_queue_name: str = "deafrica-pds-sentinel-2-sync-scene",
+    limit: int = None,
     slack_url: str = None,
 ):
     """
     Publish missing scenes
+    :param idx: (int) sequential index which will be used to define the range of scenes that the POD will work with
+    :param max_workers: (int) total number of pods used for the task. This number is used to split the number of scenes
+    equally among the PODS
+    :param sync_queue_name: (str) Sync queue name
+    :param limit: (str) optional limit of messages to be read from the report
+    :param slack_url: (str) Slack notification channel hook URL
```

Comment:
This function won't be called manually, so adding docs here is not so helpful.

Reply:
This will be there as per documentation:
https://click.palletsprojects.com/en/8.0.x/documentation/#documenting-arguments